lua/nvim-treesitter/parsers.lua

```diff
@@ -323,7 +323,16 @@ list.elixir = {
     files = { "src/parser.c", "src/scanner.cc" },
     branch = "main",
   },
-  maintainers = { "@jonatanklosko" },
+  maintainers = { "@jonatanklosko", "@connorlay" },
+}
+
+list.gleam = {
+  install_info = {
+    url = "https://github.com/J3RN/tree-sitter-gleam",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
+  maintainers = { "@J3RN", "@connorlay" },
```

Comment:
@connorlay does J3RN also contribute to nvim-treesitter? Otherwise, he should be removed from the maintainers as I and user otherwise will unnecessarily ping him when there are nvim-treesitter issues

Reply:
otherwise this PR is fine

Reply:
No J3RN is the maintainer of the tree-sitter-gleam parser itself, but not the nvim-treesitter integration.